Mage/MessageStream/MessageStream.cs

```diff
@@ -39,44 +39,51 @@ private static Logger Logger
 
 
 		//
-		private void InitializeMessageList()
+		private void Initialize()
```

Comment:
small detail, but semantically doing an initialize in dispose seems... counter-intuitive
would recommend renaming to something like ResetMessageQueue